ovos_local_backend/configuration.py

```diff
@@ -59,18 +59,78 @@
     },
     "default_ww": "hey_mycroft",  # needs to be present below
     "ww_configs": {  # these can be exposed in a web UI for selection
-        "hey_mycroft": {"phonemes": "HH EY . M AY K R AO F T",
-                        "module": "ovos-ww-plugin-pocketsphinx",
-                        "threshold": 1e-90},
-        "hey_jarvis": {"phonemes": "HH EY . JH AA R V AH S .",
```

Comment:
can you please leave the existing pocketsphinx wws? those dont have an equivalent precise model and should remain available

Reply:
Can we change them to vosk instead? It works WAY better

Reply:
Sure, sounds like a plan! Any other words we want there? Please also remove the engine from the ww name, that should be added in UI directly. The backend manager can add the module itaelf at display time. Im not sure if downstream code will like the parenthesis